.github/workflows/sonarqube.yml

```
@@ -12,16 +12,86 @@ jobs:
+          pwd
+          ls
+            
+
       - name: SonarCloud Scan
         uses: SonarSource/sonarcloud-github-action@v2
         with:
           projectBaseDir: .
-        env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+        # SONAR_TOKEN is already set in the job's env block
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)